lib/propolis/src/intr_pins.rs

```diff
@@ -188,6 +212,13 @@ impl IntrPin for FuncPin {
         let inner = self.0.lock().unwrap();
         inner.level
     }
+    fn import_state(&self, is_asserted: bool) {
+        let mut inner = self.0.lock().unwrap();
+        if inner.level != is_asserted {
+            inner.level = is_asserted;
+            (inner.func)(inner.level);
+        }
+    }
```

Comment:
I'm not certain we want to call the underlying function here?  Do we want the edge to be visible while importing state?

Reply:
I wasn't sure what to do for `FuncPin`, since it's not actually exercised in this change...I'll look into the uses of it and see what their functions actually do.